nvme_gw_server.py

```diff
@@ -18,6 +20,12 @@
 import argparse
 import json
 
+libc = ctypes.CDLL("libc.so.6")
```

Comment:
Not that the SONAME is likely to be bumped in the observable feature, but still:
```suggestion
libc = ctypes.CDLL(ctypes.util.find_library("c"))
```